src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-list/multi-cluster-list.component.ts

```diff
@@ -273,4 +273,9 @@ export class MultiClusterListComponent extends ListWithDetails implements OnInit
     super.setExpandedRow(expandedRow);
     this.router.navigate(['performance-details'], { relativeTo: this.route });
   }
+
+  refresh() {
+    this.multiClusterService.refresh();
```

Comment:
the bad thing about this is that every 5 sec we are hitting the monDB to fetch the config options, which may not be a good thing to do.. Do we need to fetch the config every 5 sec? it is not going to change unless we change it in the UI isn't it?

i think we just need to get the status of the token at regular interval to make the table up-to-date right?

Reply:
Thanks for explaining behind the scene scenario. @nizamial09 

@aaSharma14 lets improve this , I think it will be nice to address.

Thanks